# Request
# 使用 -1.0 表示不控制該項
float64 target_front_distance
float64 target_left_distance
float64 target_right_distance
float64 target_rear_distance

# -1.0 表示不進行旋轉
# 0.0 表示對齊牆面
# 其他值表示旋轉特定角度
float64 target_angle

# 當 target_angle 為 0.0 時，指定要對齊的牆面 ("front", "left", "right","rear")
# 其他情況下此欄位無效，可為空字串 ""
string align_to_wall
bool use_odometry
---
# Response
bool success
string message
